.gitignore

```diff
@@ -1 +1,3 @@
 node_modules/
+lib/
```

Comment:
Correct, then we have a `.npmignore` file which does not ignore the lib folder when publishing to npm.